### PR TITLE
Update axios to 1.7.7 and bump version to 3.1.0

### PR DESCRIPTION
**Pull request by eithe**
Ref. #323

This PR updates axios package to 1.7.7 to address some vulnerabilities in axios.

https://github.com/advisories/GHSA-jchw-25xp-jwwc
https://github.com/advisories/GHSA-8hc4-vh64-cxmj

I couldn't get the ava tests to run locally unfortunately (but #314) and I've only tested this inside my own RN project with only basic usage of apisauce so no guarantees here.

@jamonholmgren Thanks for taking my request at the React Universe conferanse. Would be cool if you guys could spend a few hours to update this project one day :)

### CODE REVIEW

**Comment by eithe** [comment]
@jamonholmgren  You promised @ React Universe ❤️

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "3.0.1",
+  "version": "3.1.0",
```
Comment (jamonholmgren) [inline package.json:2]: We bump versions automatically, so I'll take this out.

```suggestion
  "version": "3.0.1",
```

**Comment by jamonholmgren** [comment]
Semantic release didn't work; we'll look into this asap

**Comment by eithe** [comment]
@jamonholmgren Any news?

**Pull request merged**